app/javascript/react/components/molecules/Calendar/Calendar.style.tsx

```diff
@@ -1,10 +1,12 @@
 import styled from "styled-components";
 import { media } from "../../../utils/media";
 import { white } from "../../../assets/styles/colors";
+import { H3 } from "../../Typography";
 
 const ThreeMonths = styled.div`
   display: flex;
   justify-content: center;
+  align-items: center;
```

Comment:
this` align-items: center` messes with the calendar view. Without it, the calendar looks better.